ably/websocket.go

```diff
@@ -60,6 +60,7 @@ func dialWebsocket(proto string, u *url.URL, timeout time.Duration) (*websocketC
 // dialWebsocketTimeout dials the websocket with a timeout.
 func dialWebsocketTimeout(uri, protocol, origin string, timeout time.Duration) (*websocket.Conn, error) {
 	config, err := websocket.NewConfig(uri, origin)
+	config.Header.Set(ablyAgentHeader, getAblyAgentIdentifier())
```

Comment:
This should come after the error check in case `config` is nil when an error is returned.

Reply:
Done